pages/common/pageres.md

```diff
@@ -0,0 +1,29 @@
+# pageres
+
+> Capture screenshots of websites in various resolutions.
+> More information: <https://github.com/sindresorhus/pageres-cli>.
+> More information on API: <https://github.com/sindresorhus/pageres>.
+
+- Take multiple screenshots of multiple URLs at more than 1 resolution:
+
+`pageres {{https://example.com/}} {{https://example2.com/}} {{1366x768}} {{1600x900}}`
+
+- Give specific option for each specific URL, Overriding outer option:
```

Comment:
Provide specific options for a URL, overriding global options.

Let's also reduce the no. of resolutions to 1, to clarify the usage of option overriding.